integration-tests/sr25519-verification/lib.rs

```diff
@@ -0,0 +1,128 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[ink::contract]
+pub mod sr25519_verification {
+    #[ink(storage)]
+    #[derive(Default)]
+    pub struct Sr25519Verification {}
+
+    impl Sr25519Verification {
+        /// Creates a new sr25519 verification smart contract initialized with the given
+        /// value.
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            Self {}
+        }
+
+        #[ink(message)]
+        pub fn noop(&self) {}
```

Comment:
I presume since you must have a `message` in the ink! contacts, but you can't use `sr25519_verify` on the `substrate-contracts-node`, you left the function blank. Can you please either document it somehow, or double check that you can't use `sr25519_verify` with the `substrate-contracts-node`?